.github/workflows/build.yml

```diff
@@ -98,7 +98,7 @@ jobs:
 
   test:
     needs: build
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
You can leave other values as they are (`ubuntu-latest`) if there's no specific reason to change.